File: src/main/java/com/kingdom/model/computer/BigMoneyComputerPlayer.java
package com.kingdom.model.computer;

import com.kingdom.model.Card;
import com.kingdom.model.Game;
import com.kingdom.model.Player;

public class BigMoneyComputerPlayer extends HardComputerPlayer {
    public BigMoneyComputerPlayer(Player player, Game game) {
        super(player, game);
        bigMoneyUltimate = true;
    }

    @Override
    public Card buyCard() {
        return buyCardBigMoneyUltimate();
    }
}

File: src/main/kotlin/com/kingdom/model/computer/BigMoneyComputerPlayer.kt
package com.kingdom.model.computer

import com.kingdom.model.cards.Card
import com.kingdom.model.OldGame
import com.kingdom.model.OldPlayer

class BigMoneyComputerPlayer(player: OldPlayer, game: OldGame) : HardComputerPlayer(player, game) {
    init {
        isBigMoneyUltimate = true
    }

    override fun buyCard(): Card? {
        return buyCardBigMoneyUltimate()
    }
}
